src/lib.rs

```diff
@@ -7,21 +7,25 @@
 //! ```rust
 //! use thread_priority::*;
 //!
-//! assert!(set_current_thread_priority(ThreadPriority::Min).is_ok());
+//! let result = set_current_thread_priority(ThreadPriority::Min);
```

Comment:
Can I ask why did you remove the assertion? This was an example and a doc-test which was running and which was testing the library...

Reply:
On macOS it blows an error because the implementation there is a stub

Reply:
This is the pattern which works in my application. "optional" thread priority change.

```
// Ignore the response- we just keep going even if priorty can not be boosted on a given platform
    let _ = thread_priority::set_current_thread_priority(thread_priority::ThreadPriority::Max);
```

So you can pay attention to whether it is implemented for the current OS, but as an app developer you don't have to.